database-handler.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+class DatabaseHandler
+{
+    private $table_name;
+
+    public function init()
+    {
+        global $wpdb;
+
+        $this->table_name = $wpdb->prefix . "lightning_publisher_payments";
+
+        $charset_collate = $wpdb->get_charset_collate();
+
+        $sql = "CREATE TABLE $this->table_name (
+        id bigint(20) NOT NULL AUTO_INCREMENT,
+        post_id bigint(20) NOT NULL,
+        payment_hash varchar(256) NOT NULL,
+        payment_request varchar(256) NOT NULL,
+        amount_in_satoshi double(30, 20) DEFAULT 0.0 NOT NULL,
+        exchange_rate double(30, 20) DEFAULT 0.0 NOT NULL,
```

Comment:
I think we should use integers here and always store amounts in the smallest amount possible. what do you think?

Reply:
Aah ok, I see so. In case, we have floating points we can multiply them with something like `100`

Reply:
yep, exactly. 👍

Reply:
Just made the changes

Reply:
and we actually should add an index on the `payment_hash` and `post_id`

Reply:
is the `post_id` unique for invoices?

Reply:
no. one post_id has many invoices. 
whenever somebody pays for a post it will create one invoice. 

both indexes should not be unique.